From 9ee2a5b30901e57d10faeb23a7fbea9ffc6ddbf1 Mon Sep 17 00:00:00 2001
From: Sukhbir Singh <sukhbir@torproject.org>
Date: Mon, 10 Oct 2016 19:24:09 -0700
Subject: [PATCH 06/22] Remove option for configuring account picture

---
 im/content/blist.xul | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/im/content/blist.xul b/im/content/blist.xul
index 843f1eef57..71ee2c9ef8 100644
--- a/im/content/blist.xul
+++ b/im/content/blist.xul
@@ -114,8 +114,7 @@
       <stack id="statusImageStack">
         <!-- The box around the user icon is a workaround for bug 955673. -->
         <box id="userIconHolder">
-          <image id="userIcon" role="button" popup="changeUserIconPanel"
-                 aria-label="&userIcon.label;" tooltiptext="&userIcon.label;"/>
+          <image id="userIcon" role="button"/>
         </box>
         <panel id="changeUserIconPanel"
                type="arrow" align="center"
-- 
2.14.2

