From 75f578f9833babab49cccdd5d6496754c8d0c94c Mon Sep 17 00:00:00 2001
From: Sukhbir Singh <sukhbir@torproject.org>
Date: Wed, 26 Jul 2017 16:16:09 -0400
Subject: [PATCH 07/22] Modify top protocol defaults

---
 im/content/accountWizard.xul | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/im/content/accountWizard.xul b/im/content/accountWizard.xul
index 135b68386c..356321dbcc 100644
--- a/im/content/accountWizard.xul
+++ b/im/content/accountWizard.xul
@@ -51,7 +51,7 @@
     <listbox flex="1" id="protolist"
              ondblclick="document.getElementById('accountWizard').advance();"/>
     <hbox pack="end">
-      <label id="getMoreProtocols" class="text-link" value="&accountProtocolGetMore.label;"
+      <label id="getMoreProtocols" class="text-link" value=""
              onclick="if (event.button == 0) { accountWizard.openURL(this.getAttribute('getMoreURL')); }"/>
     </hbox>
   </wizardpage>
-- 
2.14.3

