From 0e9be1b8cbc5a83deea99e27db548a5ee7f27595 Mon Sep 17 00:00:00 2001
From: Sukhbir Singh <sukhbir@torproject.org>
Date: Mon, 10 Oct 2016 19:24:09 -0700
Subject: [PATCH 12/26] Account picture

---
 im/content/blist.xul | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/im/content/blist.xul b/im/content/blist.xul
index b90fddadb..f29a48b99 100644
--- a/im/content/blist.xul
+++ b/im/content/blist.xul
@@ -114,8 +114,7 @@
       <stack id="statusImageStack">
         <!-- The box around the user icon is a workaround for bug 955673. -->
         <box id="userIconHolder">
-          <image id="userIcon" role="button" popup="changeUserIconPanel"
-                 aria-label="&userIcon.label;" tooltiptext="&userIcon.label;"/>
+          <image id="userIcon" role="button"/>
         </box>
         <panel id="changeUserIconPanel"
                type="arrow" align="center"
-- 
2.11.0

