diff --git a/im/content/jar.mn b/im/content/jar.mn
--- a/im/content/jar.mn
+++ b/im/content/jar.mn
@@ -10,6 +10,8 @@
 #endif
        content/instantbird/aboutDialog.css
 *      content/instantbird/aboutDialog.xul
+       content/instantbird/aboutDialog.js
+       content/instantbird/aboutDialog-appUpdater.js
        content/instantbird/aboutPanel.xml
        content/instantbird/account.js
        content/instantbird/accounts.css
