diff --git a/im/content/blist.xul b/im/content/blist.xul
--- a/im/content/blist.xul
+++ b/im/content/blist.xul
@@ -114,8 +114,7 @@
       <stack id="statusImageStack">
         <!-- The box around the user icon is a workaround for bug 955673. -->
         <box id="userIconHolder">
-          <image id="userIcon" role="button" popup="changeUserIconPanel"
-                 aria-label="&userIcon.label;" tooltiptext="&userIcon.label;"/>
+          <image id="userIcon" role="button"/>
         </box>
         <panel id="changeUserIconPanel"
                type="arrow" align="center"
