diff --git a/im/content/blist.xul b/im/content/blist.xul
--- a/im/content/blist.xul
+++ b/im/content/blist.xul
@@ -109,18 +109,17 @@
     </menupopup>
   </popupset>
 
   <toolbox id="mainToolbox">
     <toolbar id="statusArea">
       <stack id="statusImageStack">
         <!-- The box around the user icon is a workaround for bug 955673. -->
         <box id="userIconHolder">
-          <image id="userIcon" role="button" popup="changeUserIconPanel"
-                 aria-label="&userIcon.label;" tooltiptext="&userIcon.label;"/>
+          <image id="userIcon" role="button"/>
         </box>
         <panel id="changeUserIconPanel"
                type="arrow" align="center"
                onpopupshowing="buddyList.updateUserIconPanelItems();"
                onpopuphiding="buddyList.stopWebcamStream();">
           <deck id="userIconPanel">
             <vbox class="userIconPanelMenu">
               <stack id="userIconPanelStack" flex="1">
