diff --git a/im/app/profile/moz.build b/im/app/profile/moz.build
--- a/im/app/profile/moz.build
+++ b/im/app/profile/moz.build
@@ -9,6 +9,7 @@
     DEFINES['HAVE_SHELL_SERVICE'] = 1
 
 FINAL_TARGET_FILES.defaults.profile += [
+    'cert_override.txt',
     'localstore.rdf',
     'mimeTypes.rdf',
 ]
diff --git a/im/installer/package-manifest.in b/im/installer/package-manifest.in
--- a/im/installer/package-manifest.in
+++ b/im/installer/package-manifest.in
@@ -160,6 +160,7 @@
 @RESPATH@/defaults/profile/localstore.rdf
 @RESPATH@/defaults/profile/prefs.js
 @RESPATH@/defaults/profile/mimeTypes.rdf
+@RESPATH@/defaults/profile/cert_override.txt
 
 #ifdef XP_MACOSX
 @RESPATH@/components/ibDockBadge.js
diff --git a/im/installer/Makefile.in b/im/installer/Makefile.in
--- a/im/installer/Makefile.in
+++ b/im/installer/Makefile.in
@@ -109,7 +109,9 @@
 MOZ_PKG_MAC_EXTRA=--symlink "/Applications:/ "
 endif
 
-NON_OMNIJAR_FILES =
+NON_OMNIJAR_FILES = \
+       defaults/profile/cert_override.txt \
+       $(NULL)
 
 INSTALL_SDK = 1
 
