diff -r 6300e54fef35 chat/chat-prefs.js
--- a/chat/chat-prefs.js	Wed Oct 15 20:21:36 2014 -0700
+++ b/chat/chat-prefs.js	Thu Oct 16 07:15:08 2014 -0400
@@ -6,7 +6,7 @@
 //  0 = do not connect / show the account manager
 //  1 = connect automatically
 //  Other values will be added later, for example to start minimized
-pref("messenger.startup.action", 1);
+// pref("messenger.startup.action", 1);
 
 pref("messenger.accounts", "");
 
@@ -106,7 +106,50 @@
 pref("purple.debug.loglevel", 2);
 #endif
 
+/* Tor Messenger */
+// Logging
+// Disable all logging
 pref("purple.logging.format", "json");
-pref("purple.logging.log_chats", true);
-pref("purple.logging.log_ims", true);
-pref("purple.logging.log_system", true);
+pref("purple.logging.log_chats", false);
+pref("purple.logging.log_ims", false);
+pref("purple.logging.log_system", false);
+
+// Network
+// Use a manual proxy configuration
+pref("network.proxy.type", 1);
+// Configure Instantbird to use the SOCKS5 proxy
+pref("network.proxy.socks", "127.0.0.1");
+pref("network.proxy.socks_port", 9150);
+pref("network.proxy.socks_version", 5);
+// Set DNS proxying through SOCKS5
+pref("network.proxy.socks_remote_dns", true);
+// Warn when an external application is to be launched
+pref("network.protocol-handler.warn-external.http", true);
+pref("network.protocol-handler.warn-external.https", true);
+pref("network.protocol-handler.warn-external.ftp", true);
+pref("network.protocol-handler.warn-external.file", true);
+pref("network.protocol-handler.warn-external-default", true);
+
+// Security
+// Disable SSLv3 by setting the minimum supported protocol to TLS 1.0.
+pref("security.tls.version.min", 1);
+// Disable geolocation
+pref("geo.enabled", false),
+// Empty the user agent
+pref("general.useragent.extra.instantbird", "");
+
+// Messenger
+//  0 = do not connect / show the account manager
+pref("messenger.startup.action", 0);
+// Do not report idle status or the away message
+pref("messenger.status.awayWhenIdle", false);
+pref("messenger.status.defaultIdleAwayMessage", "");
+pref("messenger.status.reportIdle", false);
+
+// Browser
+// Disable caching
+pref("browser.cache.disk.enable", false);
+pref("browser.cache.offline.enable", false);
+
+// IRC
+pref("chat.irc.defaultQuitMessage", "");
diff -r 6300e54fef35 im/app/profile/all-instantbird.js
--- a/im/app/profile/all-instantbird.js	Wed Oct 15 20:21:36 2014 -0700
+++ b/im/app/profile/all-instantbird.js	Thu Oct 16 07:15:08 2014 -0400
@@ -309,7 +309,7 @@
 
 // Comma separated list of prpl ids that should use libpurple even if there is
 // a JS implementation. This is used to land JS-prpls pref'ed off in nightlies.
-pref("chat.prpls.forcePurple", "prpl-jabber");
+pref("chat.prpls.forcePurple", "");
 
 // Whether to parse log files for conversation statistics.
 pref("statsService.parseLogsForStats", true);
