diff --git a/chat/protocols/irc/ircCTCP.jsm b/chat/protocols/irc/ircCTCP.jsm
index 5722479..6b82caa 100644
--- a/chat/protocols/irc/ircCTCP.jsm
+++ b/chat/protocols/irc/ircCTCP.jsm
@@ -195,7 +195,7 @@ var ctcpBase = {
       if (aMessage.command == "PRIVMSG") {
         // TIME
         // Received a TIME request, send a human readable response.
-        let now = (new Date()).toString();
+        let now = (new Date()).toUTCString();
         this.LOG("Received TIME request from " + aMessage.origin +
                  ". Sending TIME response: \"" + now + "\".");
         this.sendCTCPMessage(aMessage.origin, true, "TIME", ":" + now);
