diff --git a/im/installer/package-manifest.in b/im/installer/package-manifest.in
--- a/im/installer/package-manifest.in
+++ b/im/installer/package-manifest.in
@@ -104,6 +104,8 @@
 #endif
 
 #endif # MOZ_ANGLE_RENDERER
+
+#ifdef USE_ICU
 #ifndef MOZ_NATIVE_ICU
 #ifdef MOZ_SHARED_ICU
 #ifdef XP_WIN
@@ -121,6 +123,7 @@
 #endif
 #endif
 #endif
+#endif
 
 @RESPATH@/components/PACGenerator.js
 @RESPATH@/components/PACGenerator.manifest
