diff --git a/im/content/accountWizard.xul b/im/content/accountWizard.xul
--- a/im/content/accountWizard.xul
+++ b/im/content/accountWizard.xul
@@ -50,7 +50,7 @@
     <listbox flex="1" id="protolist"
              ondblclick="document.getElementById('accountWizard').advance();"/>
     <hbox pack="end">
-      <label id="getMoreProtocols" class="text-link" value="&accountProtocolGetMore.label;"
+      <label id="getMoreProtocols" class="text-link" value=""
              onclick="if (event.button == 0) { accountWizard.openURL(this.getAttribute('getMoreURL')); }"/>
     </hbox>
   </wizardpage>
