diff --git a/chat/protocols/irc/irc.js b/chat/protocols/irc/irc.js
index e5be455..533be08 100644
--- a/chat/protocols/irc/irc.js
+++ b/chat/protocols/irc/irc.js
@@ -1833,13 +1833,13 @@ ircProtocol.prototype = {
 
   usernameSplits: [
     {get label() _("options.server"), separator: "@",
-     defaultValue: "chat.freenode.net", reverse: true}
+     defaultValue: "irc.oftc.net", reverse: true}
   ],
 
   options: {
     // TODO Default to IRC over SSL.
-    "port": {get label() _("options.port"), default: 6667},
-    "ssl": {get label() _("options.ssl"), default: false},
+    "port": {get label() _("options.port"), default: 6697},
+    "ssl": {get label() _("options.ssl"), default: true},
     // TODO We should attempt to auto-detect encoding instead.
     "encoding": {get label() _("options.encoding"), default: "UTF-8"},
     "quitmsg": {get label() _("options.quitMessage"),
