diff --git a/chat/protocols/irc/irc.js b/chat/protocols/irc/irc.js
index c581e2d..cf0d085 100644
--- a/chat/protocols/irc/irc.js
+++ b/chat/protocols/irc/irc.js
@@ -1916,7 +1916,7 @@ ircProtocol.prototype = {
 
   usernameSplits: [
     {get label() _("options.server"), separator: "@",
-     defaultValue: "chat.freenode.net", reverse: true}
+     defaultValue: "", reverse: true}
   ],
 
   options: {
