diff --git a/im/app/profile/all-instantbird.js b/im/app/profile/all-instantbird.js
--- a/im/app/profile/all-instantbird.js
+++ b/im/app/profile/all-instantbird.js
@@ -28,7 +28,7 @@
 // 0 = spellcheck nothing
 // 1 = check multi-line controls [default]
 // 2 = check multi/single line controls
-pref("layout.spellcheckDefault", 1);
+pref("layout.spellcheckDefault", 0);
 
 pref("messenger.accounts.convertOldPasswords", true);
 pref("messenger.accounts.promptOnDelete", true);
@@ -66,18 +66,18 @@
 
 // Whether message related sounds should be played at all. If this is enabled
 // then the more specific prefs are checked as well.
-pref("messenger.options.playSounds.message", true);
+pref("messenger.options.playSounds.message", false);
 // Specifies whether each message event should trigger a sound for incoming
 // and outgoing messages, or when your nickname is mentioned in a chat.
-pref("messenger.options.playSounds.outgoing", true);
-pref("messenger.options.playSounds.incoming", true);
-pref("messenger.options.playSounds.alert", true);
+pref("messenger.options.playSounds.outgoing", false);
+pref("messenger.options.playSounds.incoming", false);
+pref("messenger.options.playSounds.alert", false);
 // Whether contact list related sounds should be played at all. If this is
 // enabled then the more specific prefs are checked as well.
 pref("messenger.options.playSounds.blist", false);
 // Specifies whether sounds should be played on login/logout events.
-pref("messenger.options.playSounds.login", true);
-pref("messenger.options.playSounds.logout", true);
+pref("messenger.options.playSounds.login", false);
+pref("messenger.options.playSounds.logout", false);
 
 pref("font.default.x-western", "sans-serif");
 pref("font.default.x-unicode", "sans-serif");
@@ -142,26 +142,28 @@
 
 // Update service URL:
 // You do not need to use all the %VAR% parameters. Use what you need, %PRODUCT%,%VERSION%,%BUILD_ID%,%CHANNEL% for example
-pref("app.update.url", "https://update.instantbird.org/1/%PRODUCT%/%VERSION%/%BUILD_ID%/%BUILD_TARGET%/%LOCALE%/%CHANNEL%/%OS_VERSION%/update.xml");
+pref("app.update.url", "https://www.torproject.org/dist/tormessenger/update_2/%CHANNEL%/%BUILD_TARGET%/%VERSION%/%LOCALE%");
+
+#ifdef XP_WIN
+// For now, disable staged updates on Windows (see #18292).
+pref("app.update.staging.enabled", false);
+#endif
 
 // URL user can browse to manually if for some reason all update installation
 // attempts fail.
-pref("app.update.url.manual", "http://www.instantbird.com/download.html");
+pref("app.update.url.manual", "https://www.torproject.org");
 
 // A default value for the "More information about this update" link
 // supplied in the "An update is available" page of the update wizard.
-pref("app.update.url.details", "http://www.instantbird.com/");
-
-// User-settable override to app.update.url for testing purposes.
-//pref("app.update.url.override", "");
+pref("app.update.url.details", "https://trac.torproject.org/projects/tor/wiki/doc/TorMessenger");
 
 // Interval: Time between checks for a new version (in seconds)
 //           default=1 day
-pref("app.update.interval", 86400);
+pref("app.update.interval", 43200);
 
 // Interval: Time before prompting the user to download a new version that
 //           is available (in seconds) default=1 day
-pref("app.update.nagTimer.download", 86400);
+pref("app.update.nagTimer.download", 3600);
 
 // Interval: Time before prompting the user to restart to install the latest
 //           download (in seconds) default=30 minutes
@@ -202,7 +204,7 @@
 pref("browser.search.order.2",                "chrome://instantbird/locale/region.properties");
 
 // send ping to the server to update
-pref("browser.search.update", true);
+pref("browser.search.update", false);
 
 // disable logging for the search service update system by default
 pref("browser.search.update.log", false);
@@ -245,9 +247,9 @@
 pref("extensions.getMoreProtocolsURL", "https://add-ons.instantbird.org/%LOCALE%/%APP%/%VERSION%/protocols/");
 
 // suppress external-load warning for standard browser schemes
-pref("network.protocol-handler.warn-external.http", false);
-pref("network.protocol-handler.warn-external.https", false);
-pref("network.protocol-handler.warn-external.ftp", false);
+pref("network.protocol-handler.warn-external.http", true);
+pref("network.protocol-handler.warn-external.https", true);
+pref("network.protocol-handler.warn-external.ftp", true);
 
 // don't load links inside Instantbird
 pref("network.protocol-handler.expose-all", false);
@@ -262,10 +264,10 @@
 pref("network.protocol-handler.expose.javascript", true);
 
 // 0-Accept, 1-dontAcceptForeign, 2-dontUse
-pref("network.cookie.cookieBehavior", 0);
+pref("network.cookie.cookieBehavior", 1);
 
 // The breakpad report server to link to in about:crashes
-pref("breakpad.reportURL", "http://crash-stats.instantbird.com/report/index/");
+pref("breakpad.reportURL", "https://crash-stats.instantbird.com/report/index/");
 
 // We have an Error Console menu item by default so let's display chrome errors
 pref("javascript.options.showInConsole", true);
@@ -300,14 +302,70 @@
 // 3  at the end of the tabstrip
 pref("browser.tabs.closeButtons", 1);
 
-#expand pref("chat.irc.defaultQuitMessage", "Instantbird __APP_VERSION__ -- http://www.instantbird.com");
+#expand pref("chat.irc.defaultQuitMessage", "");
 
 pref("chat.twitter.consumerKey", "TSuyS1ieRAkB3qWv8yyEw");
 pref("chat.twitter.consumerSecret", "DKtKaSf5a7pBNhdBsSZHTnI5Y03hRlPFYWmb4xXBlkU");
 
 // Comma separated list of prpl ids that should use libpurple even if there is
 // a JS implementation. This is used to land JS-prpls pref'ed off in nightlies.
-pref("chat.prpls.forcePurple", "prpl-jabber");
+pref("chat.prpls.forcePurple", "");
 
 // Whether to parse log files for conversation statistics.
-pref("statsService.parseLogsForStats", true);
+pref("statsService.parseLogsForStats", false);
+
+/* Tor Messenger */
+// Logging
+// Disable all logging
+pref("purple.logging.log_chats", false);
+pref("purple.logging.log_ims", false);
+pref("purple.logging.log_system", false);
+
+// Network
+// Use a manual proxy configuration
+pref("network.proxy.type", 1);
+// Empty the "no proxy" setting
+pref("network.proxy.no_proxies_on", "");
+// Configure Instantbird to use the SOCKS5 proxy
+pref("network.proxy.socks", "127.0.0.1");
+pref("network.proxy.socks_port", 9152);
+pref("network.proxy.socks_version", 5);
+// Set DNS proxying through SOCKS5
+pref("network.proxy.socks_remote_dns", true);
+// Disable DNS prefetching
+pref("network.dns.disablePrefetch", true);
+// Disable SPDY
+pref("network.http.spdy.enabled", false);
+// Set the user-agent to Instantbird stable
+pref("general.useragent.override", "Mozilla/5.0 (Windows NT 6.1; rv:25.0) Gecko/20100101 Instantbird/1.5");
+
+// Security
+// Disable SSLv3 by setting the minimum supported protocol to TLS 1.0.
+pref("security.tls.version.min", 1);
+// Disable geolocation
+pref("geo.enabled", false);
+
+// Do not report idle status or the away message
+pref("messenger.status.awayWhenIdle", false);
+pref("messenger.status.defaultIdleAwayMessage", "");
+pref("messenger.status.reportIdle", false);
+// Do not send the message format (fonts, colors)
+pref("messenger.conversations.sendFormat", false);
+// Disable text formatting (remove the tags)
+pref("messenger.options.filterMode", 0);
+// Disable typing notifications
+pref("purple.conversations.im.send_typing", false);
+
+// Browser
+// Disable caching
+pref("browser.cache.disk.enable", false);
+pref("browser.cache.offline.enable", false);
+
+// Media
+// Disable WebRTC
+pref("media.peerconnection.enabled", false);
+// Disable "Take Picture" functionality that accesses the webcam
+pref("media.navigator.video.enabled", false);
+// Disable hardware acceleration
+pref("gfx.direct2d.disabled", true);
+pref("layers.acceleration.disabled", true);
