diff --git a/im/content/nsContextMenu.js b/im/content/nsContextMenu.js
--- a/im/content/nsContextMenu.js
+++ b/im/content/nsContextMenu.js
@@ -462,33 +462,17 @@ nsContextMenu.prototype = {
     var selectedText = getBrowserSelection(16);
 
     if (!selectedText)
       return false;
 
     if (selectedText.length > 15)
       selectedText = selectedText.substr(0,15) + this.ellipsis;
 
-    var engine = Services.search.defaultEngine;
-    if (!engine)
-      return false;
-
-    // format "Search <engine> for <selection>" string to show in menu
-    var bundle = document.getElementById("bundle_instantbird");
-    var menuLabel = bundle.getFormattedString("contextMenuSearchText",
-                                              [engine.name,
-                                               selectedText]);
-    document.getElementById("context-searchselect").label = menuLabel;
-    document.getElementById("context-searchselect").accessKey =
-      bundle.getString("contextMenuSearchText.accesskey");
-    menuLabel = bundle.getFormattedString("contextMenuSearchWith",
-                                          [selectedText]);
-    document.getElementById("context-searchselect-with").label = menuLabel;
-
-    return true;
+    return false;
   },
 
   // Returns true if anything is selected.
   isContentSelection: function() {
     return !document.commandDispatcher.focusedWindow.getSelection().isCollapsed;
   }
 };
 
