diff --git a/chat/protocols/xmpp/xmpp.js b/chat/protocols/xmpp/xmpp.js
index f99a238..bc787c9 100644
--- a/chat/protocols/xmpp/xmpp.js
+++ b/chat/protocols/xmpp/xmpp.js
@@ -29,12 +29,12 @@ XMPPProtocol.prototype = {
 
   usernameSplits: [
     {get label() _("options.domain"), separator: "@",
-     defaultValue: "jabber.org", reverse: true}
+     defaultValue: "jabber.ccc.de", reverse: true}
   ],
 
   options: {
     resource: {get label() _("options.resource"),
-               get default() XMPPDefaultResource},
+               default: ""},
     priority: {get label() _("options.priority"), default: 0},
     connection_security: {
       get label() _("options.connectionSecurity"),
@@ -47,7 +47,7 @@ XMPPProtocol.prototype = {
       },
       default: "require_tls"
     },
-    server: {get label() _("options.connectServer"), default: ""},
+    server: {get label() _("options.connectServer"), default: "jabber.ccc.de"},
     port: {get label() _("options.connectPort"), default: 5222}
   },
   get chatHasTopic() true,
