diff --git a/chat/protocols/xmpp/xmpp.js b/chat/protocols/xmpp/xmpp.js
--- a/chat/protocols/xmpp/xmpp.js
+++ b/chat/protocols/xmpp/xmpp.js
@@ -24,17 +24,17 @@ XMPPProtocol.prototype = {
   __proto__: GenericProtocolPrototype,
   get normalizedName() "jabber",
   get name() "XMPP",
   get iconBaseURI() "chrome://prpl-jabber/skin/",
   getAccount: function(aImAccount) new XMPPAccount(this, aImAccount),
 
   usernameSplits: [
     {get label() _("options.domain"), separator: "@",
-     defaultValue: "jabber.org", reverse: true}
+     defaultValue: "", reverse: true}
   ],
 
   options: {
     resource: {get label() _("options.resource"),
                get default() XMPPDefaultResource},
     priority: {get label() _("options.priority"), default: 0},
     connection_security: {
       get label() _("options.connectionSecurity"),
