diff --git a/im/content/accountWizard.js b/im/content/accountWizard.js
index 707f1c7..2ca929a 100644
--- a/im/content/accountWizard.js
+++ b/im/content/accountWizard.js
@@ -7,6 +7,14 @@ Cu.import("resource:///modules/imServices.jsm");
 
 const PREF_EXTENSIONS_GETMOREPROTOCOLSURL = "extensions.getMoreProtocolsURL";
 
+// Borrowed and inspired by xmpp-client.
+const kServerOnions = {
+  "riseup.net":                "4cjw6cwpeaeppfqz.onion",
+  "jabber.ccc.de":             "okj7xc6j2szr2y75.onion",
+  "jabber.otr.im":             "5rgdtlawqkcplz75.onion",
+  "jabber.calyxinstitute.org": "ijeeynrc6x2uy5ob.onion",
+}
+
 var accountWizard = {
   onload: function aw_onload() {
     let topProtoList = document.getElementById("topprotolist");
@@ -105,6 +113,21 @@ var accountWizard = {
     return textbox;
   },
 
+  insertOnionAddress: function aw_insertOnionAddress() {
+    // Currently, we only use onion addresses for XMPP.
+    if (this.proto.id == "prpl-jabber") {
+      // If the value of the domain is one for which there exists a
+      // hidden service, replace the address with the onion address.
+      this.jabberDomain = document.getElementById("username-split-0");
+      if (this.jabberDomain.value in kServerOnions) {
+        this.onionAddress = kServerOnions[this.jabberDomain.value];
+      }
+      else {
+        this.onionAddress = "";
+      }
+    }
+  },
+
   showUsernamePage: function aw_showUsernamePage() {
     let proto = this.proto.id;
     if ("userNameBoxes" in this && this.userNameProto == proto) {
@@ -160,11 +183,20 @@ var accountWizard = {
     document.getElementById("accountusername").next = next;
   },
 
+  setOnionAddress: function aw_setOnionAddress() {
+    if (this.proto.id == "prpl-jabber") {
+      document.getElementById("prpl-jabber-server").value = this.onionAddress;
+    }
+  },
+
   showAdvanced: function aw_showAdvanced() {
     // ensure we don't destroy user data if it's not necessary
     let id = this.proto.id;
-    if ("protoSpecOptId" in this && this.protoSpecOptId == id)
+    if ("protoSpecOptId" in this && this.protoSpecOptId == id) {
+      // But we should still set the onion address, if it exists.
+      this.setOnionAddress();
       return;
+    }
     this.protoSpecOptId = id;
 
 /* FIXME
@@ -172,6 +204,7 @@ var accountWizard = {
       !this.proto.newMailNotification;
 */
     this.populateProtoSpecificBox();
+    this.setOnionAddress();
 
     let proxyVisible = this.proto.usePurpleProxy;
     if (proxyVisible) {
@@ -407,6 +440,13 @@ var accountWizard = {
       let label = bundle.getFormattedString("accountColon", [opt.opt.label]);
       rows.appendChild(this.createSummaryRow(label, opt.value));
     }
+
+    if (this.onionAddress) {
+      let onionLabel = document.getElementById("topProtocolsBundle");
+      let onion = document.getElementById("onionAddress");
+      onion.value = onionLabel.getFormattedString("onionAddress.label",
+                                  [this.jabberDomain.value, this.onionAddress]);
+    }
   },
 
   createAccount: function aw_createAccount() {
