diff --git a/chat/protocols/xmpp/xmpp.js b/chat/protocols/xmpp/xmpp.js
index b05c77e..f8c829f 100644
--- a/chat/protocols/xmpp/xmpp.js
+++ b/chat/protocols/xmpp/xmpp.js
@@ -41,7 +41,7 @@ XMPPProtocol.prototype = {
       },
       default: "require_tls"
     },
-    server: {get label() _("options.connectServer"), default: ""},
+    server: {get label() _("options.connectServer"), default: "jabber.ccc.de"},
     port: {get label() _("options.connectPort"), default: 5222}
   },
   get chatHasTopic() true,
