diff --git a/im/content/accountWizard.xul b/im/content/accountWizard.xul
index 5fa5b82..d9ce335 100644
--- a/im/content/accountWizard.xul
+++ b/im/content/accountWizard.xul
@@ -59,7 +59,8 @@
               label="&accountUsernameTitle.label;"
               onpageshow="accountWizard.showUsernamePage();"
               onpagehide="accountWizard.hideUsernamePage();"
-              onpagerewound="return accountWizard.rewindFromUsernamePage();">
+              onpagerewound="return accountWizard.rewindFromUsernamePage();"
+              onpageadvanced="return accountWizard.insertOnionAddress();">
     <description id="usernameInfo"/>
     <separator/>
     <vbox id="userNameBox"/>
@@ -98,7 +99,7 @@
     <checkbox id="newMailNotification"
               label="&accountAdvanced.newMailNotification.label;" hidden="true"/>
 
-    <groupbox id="protoSpecificGroupbox" class="collapsable" closed="true"
+    <groupbox id="protoSpecificGroupbox" class="collapsable" closed="false"
               onkeypress="accountWizard.onGroupboxKeypress(event)">
       <caption id="protoSpecificCaption"
                onclick="accountWizard.toggleGroupbox('protoSpecificGroupbox')"/>
@@ -137,6 +138,7 @@
       </columns>
       <rows id="summaryRows"/>
     </grid>
+    <description id="onionAddress" />
     <separator/>
     <checkbox id="connectAutomatically" label= "&accountSummary.connectAutomatically.label;" checked="true"/>
   </wizardpage>
