diff --git a/chat/protocols/gtalk/gtalk.js b/chat/protocols/gtalk/gtalk.js
--- a/chat/protocols/gtalk/gtalk.js
+++ b/chat/protocols/gtalk/gtalk.js
@@ -96,7 +96,7 @@
   getAccount: function(aImAccount) { return new GTalkAccount(this, aImAccount); },
   options: {
     resource: {get label() { return _("options.resource"); },
-               get default() { return XMPPDefaultResource; }}
+               default: "Instantbird"},
   },
   get chatHasTopic() { return true; },
   classID: Components.ID("{38a224c1-6748-49a9-8ab2-efc362b1000d}")
diff --git a/chat/protocols/xmpp/xmpp.js b/chat/protocols/xmpp/xmpp.js
--- a/chat/protocols/xmpp/xmpp.js
+++ b/chat/protocols/xmpp/xmpp.js
@@ -36,7 +36,7 @@
 
   options: {
     resource: {get label() { return _("options.resource"); },
-               get default() { return XMPPDefaultResource; }},
+               default: "Instanbird"},
     priority: {get label() { return _("options.priority"); }, default: 0},
     connection_security: {
       get label() { return _("options.connectionSecurity"); },
