diff --git a/src/defaults/preferences/prefs.js b/src/defaults/preferences/prefs.js
index 2c0a95e..0a0c299 100644
--- a/src/defaults/preferences/prefs.js
+++ b/src/defaults/preferences/prefs.js
@@ -1,6 +1,6 @@
 // When presenting the setup wizard, first prompt for locale.
 pref("intl.locale.matchOS", true);
-pref("extensions.torlauncher.prompt_for_locale", true);
+pref("extensions.torlauncher.prompt_for_locale", false);
 
 pref("extensions.torlauncher.loglevel", 4);  // 1=verbose, 2=debug, 3=info, 4=note, 5=warn
 pref("extensions.torlauncher.logmethod", 1);  // 0=stdout, 1=errorconsole, 2=debuglog
