diff --git a/src/install.rdf b/src/install.rdf
index 2eea493..4ce8ac8 100644
--- a/src/install.rdf
+++ b/src/install.rdf
@@ -34,7 +34,7 @@
         <em:targetApplication>
             <Description>
                 <em:id>{33cb9019-c295-46dd-be21-8c4936574bee}</em:id>
-                <em:minVersion>1.4</em:minVersion>
+                <em:minVersion>0</em:minVersion>
                 <em:maxVersion>*.*.*</em:maxVersion>
             </Description>
         </em:targetApplication>
