diff -crB binutils-2.24/bfd/peXXigen.c binutils-2.24/bfd/peXXigen.c
*** binutils-2.24/bfd/peXXigen.c	Tue Nov 26 11:37:33 2013
--- binutils-2.24/bfd/peXXigen.c	Mon Aug  4 07:10:15 2014
***************
*** 795,800 ****
--- 795,802 ----
    /* Only use a real timestamp if the option was chosen.  */
    if ((pe_data (abfd)->insert_timestamp))
      H_PUT_32 (abfd, time(0), filehdr_out->f_timdat);
+   else
+     H_PUT_32 (abfd, 0, filehdr_out->f_timdat);
  
    PUT_FILEHDR_SYMPTR (abfd, filehdr_in->f_symptr,
  		      filehdr_out->f_symptr);
