diff --git a/im/branding/messenger/branding.nsi b/im/branding/messenger/branding.nsi
index e5b5348..276722b 100755
--- a/im/branding/messenger/branding.nsi
+++ b/im/branding/messenger/branding.nsi
@@ -6,8 +6,8 @@
 
 # BrandFullNameInternal is used for some registry and file system values that
 # should not contain release that may be in the BrandFullName (e.g. Beta 1, etc.)
-!define BrandFullNameInternal "Instantbird"
-!define CompanyName           "Instantbird"
-!define URLInfoAbout          "http://www.instantbird.com/"
-!define URLUpdateInfo         "http://www.instantbird.com/"
+!define BrandFullNameInternal "Tor Messenger"
+!define CompanyName           "Tor Messenger"
+!define URLInfoAbout          "https://www.torproject.org"
+!define URLUpdateInfo         "https://www.torproject.org"
 
diff --git a/im/branding/messenger/locales/en-US/brand.dtd b/im/branding/messenger/locales/en-US/brand.dtd
index c569ebb..2d6a5d8 100644
--- a/im/branding/messenger/locales/en-US/brand.dtd
+++ b/im/branding/messenger/locales/en-US/brand.dtd
@@ -4,7 +4,7 @@
 
 <!-- nightly branding -->
 
-<!ENTITY  brandShortName        "Instantbird">
-<!ENTITY  brandFullName         "Instantbird - Nightly">
+<!ENTITY  brandShortName        "Tor Messenger">
+<!ENTITY  brandFullName         "Tor Messenger - Alpha">
 <!ENTITY  brandMotto            "'Cause geeks can also do magic!">
-<!ENTITY  vendorShortName       "Instantbird">
+<!ENTITY  vendorShortName       "Tor Messenger">
diff --git a/im/branding/messenger/locales/en-US/brand.properties b/im/branding/messenger/locales/en-US/brand.properties
index f949ced..93528a3 100644
--- a/im/branding/messenger/locales/en-US/brand.properties
+++ b/im/branding/messenger/locales/en-US/brand.properties
@@ -2,6 +2,6 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-brandShortName=Instantbird
-brandFullName=Instantbird - Nightly
-vendorShortName=Instantbird
+brandShortName=Tor Messenger
+brandFullName=Tor Messenger - Alpha
+vendorShortName=Tor Messenger
